plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLUtils.java

```diff
@@ -748,7 +748,7 @@ public static String getScriptDescription(@NotNull IFile sqlScript)
         return null;
     }
 
-    public static String generateEntityAlias(DBSEntity entity, DBRFinder<Boolean, String> aliasFinder) {
+    public static String generateEntityAlias(DBSEntity entity, boolean useUpperCase, DBRFinder<Boolean, String> aliasFinder) {
```

Comment:
You can use [`DBPIdentifierCase`](https://github.com/dbeaver/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/DBPIdentifierCase.java) that already shipped with the case transformer in it.